MovieGPT-api/src/main/java/lt/techin/group/project/rest/dto/GenreDto.java

```diff
@@ -12,7 +13,7 @@
 @AllArgsConstructor
 public class GenreDto {
 
-
+    @Schema(hidden = true)
     private Long id;
```

Comment:
Using @Schema(hidden = true) is not inherently wrong, but it's usually not the best practice unless you're hiding internal fields for very specific reasons, and the underlying logic is clearly defined elsewhere.